UnityClient/Assets/Scripts/TowerDefence/Bullet.cs

```diff
@@ -1,30 +1,31 @@
 ﻿using UnityEngine;
+using UnityEngine.Serialization;
 
 public class Bullet : MonoBehaviour
 {
-    private Transform target;
+    private Transform _target;
 
-    public float speed = 70f;
-    public int damage = 50;
-    public GameObject impactEffect;
+    [FormerlySerializedAs("speed")] [SerializeField] private float _speed = 70f;
+    [FormerlySerializedAs("damage")] [SerializeField] private int _damage = 50;
+    [FormerlySerializedAs("impactEffect")] [SerializeField] private GameObject _impactEffect;
     
     public void Seek(Transform _target)
```

Comment:
method parameters don't need an underscore prefix. Please remove it from the parameter name.